pkg/apis/resolution/v1alpha1/resolution_request_validation.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+
+	"github.com/tektoncd/resolution/pkg/common"
```

Comment:
not for this PR but at some point we should get rid of the "common" package and just have the helper function in better named packages

Reply:
Good idea - I'll do that in the PR where I move that package in.